### PR TITLE
adapt to online upgrade

### PR DESCRIPTION
**Pull request by jreidinger**
related pr: https://github.com/yast/yast-registration/pull/454

### CODE REVIEW

File: control/control.leanos.xml [hunk 1]
```diff
@@ -646,6 +649,10 @@ Please visit us at http://www.suse.com/.
                     <enable_next>yes</enable_next>
                 </module>
                 <module>
+                    <name>offline_migration</name>
+                    <enable_back>yes</enable_back>
+                </module>
+		<module>
                     <label>Product License</label>
                     <name>product_upgrade_license</name>
```
Comment (lslezak) [inline control/control.leanos.xml:657]: Hm, I think we really need to first display the license. If the user declines the license we have already upgraded the product on the SCC and modified the repository setup. The rollback is not trivial and might fail.

Displaying the license works in the manual installation, I cannot remember which changes we exactly did there but I think it should be possible also here.

File: control/control.leanos.xml [hunk 2]
```diff
@@ -646,19 +649,15 @@ Please visit us at http://www.suse.com/.
...
                     <enable_back>yes</enable_back>
                     <enable_next>yes</enable_next>
                 </module>
-                <module>
-                    <name>upgrade_urls</name>
```
Comment (lslezak) [inline control/control.leanos.xml:655]: IIRC the `upgrade_urls` allows to reusing some repositories from the old system. Registration allows adding new repositories but it's quite difficult to enter the URLs again. With this client you enable the old repos with a single click.

Comment (jreidinger) [reply control/control.leanos.xml:655]: well, it is now quite broken as it shows old repos and also shows repos from scc. which looks strange. But it does not break any stuff, just looks strange. So I can readd it

**Review by lslezak — APPROVED** [review]
LGTM

**Pull request merged**